### PR TITLE
update to version 1.6.3

### PR DESCRIPTION
**Pull request by mimmi20**

### CODE REVIEW

File: VirtualCapability/VirtualCapability.php [hunk 1]
```diff
@@ -260,6 +260,7 @@ class VirtualCapability_IsAppWebview extends VirtualCapability {
 		'MiuiBrowser',
 		'MQQBrowser',
 		'CriOS',
+	    'Firefox',
```
Comment (sebastien-fauvel) [inline VirtualCapability/VirtualCapability.php]: Formatting

File: WurflConstants.php [hunk 2]
```diff
@@ -248,5 +248,8 @@ class WurflConstants {
 		'feedfetcher-google',
 		'mowser',
 		'trove',
+	    'google web preview',
```
Comment (sebastien-fauvel) [inline WurflConstants.php]: Formatting

File: VirtualCapability/VirtualCapability.php (hunk 1 above)
Comment (mimmi20) [inline VirtualCapability/VirtualCapability.php]: I have no intension to fix the messy coding style of the code I downloaded.

Comment (sebastien-fauvel) [inline VirtualCapability/VirtualCapability.php]: Ok, no problem! I just wanted to make sure this comes from the original code & not from your own messing around ;)

**Comment by fauvel** [comment]
@mimmi20 please merge at your convenience.

**Comment by mimmi20** [comment]
merged

**Pull request merged**